tfhe/src/core_crypto/algorithms/lwe_keyswitch.rs

```diff
@@ -146,6 +150,20 @@ pub fn keyswitch_lwe_ciphertext<Scalar, KSKCont, InputCont, OutputCont>(
     // Copy the input body to the output ciphertext
     *output_lwe_ciphertext.get_mut_body().data = *input_lwe_ciphertext.get_body().data;
 
+    // If the moduli are not the same, we need to round the body in the output ciphertext
+    if output_ciphertext_modulus != input_ciphertext_modulus
+        && !output_ciphertext_modulus.is_native_modulus()
```

Comment:
Isn't there anything to do if `output_ciphertext_modulus != input_ciphertext_modulus` and `output_ciphertext_modulus.is_native_modulus()`?

Reply:
for now both moduli are power of 2, if the output modulus is the native modulus then no rounding needs to be done, as all the bits are withing the modulus